core_plugins/README.md

```diff
@@ -0,0 +1,26 @@
+# Kibana Core Plugins
+
+This folder contains all the Kibana core plugins. Each plugin is built on its
+own, so it contains a separate `package.json` and build script, including a
+separate TypeScript config — i.e. they are independent.
+
+TODO: describe dev setup or link to eventual package we build that handles it
+
+## Why are core plugins "independent"?
+
+There are several reasons why we want to keep core plugins independent, e.g.:
+
+- It forces us to treat internal and external plugins the same. They get the
+  same access to Kibana apis, they get the same build treatment, etc. That means
+  we treat external plugins as a true first-class citizen.
+- Plugins definitely don't import into core Kibana.
```

Comment:
import from the core? or export into the core?

I am not completely sure, which way is meant. Could we maybe rephrase that?

Reply:
Good input. Does it make more sense now?